docs/src/features/callback_library.md

```diff
@@ -112,7 +112,7 @@ satisfies this property mathematically it can be difficult for ODE solvers to
 ensure it numerically, as these [MATLAB examples](https://www.mathworks.com/help/matlab/math/nonnegative-ode-solution.html)
 show.
 
-In order to deal with this problem one can specify `isoutofdomain=(u,p,t) -> any(x
+In order to deal with this problem one can specify `isoutofdomain=(u,p,t) -> !any(x
```

Comment:
if it's ever true, it will reject. The original way of writing will have it reject when any term is negative.

Reply:
From the docs:
> isoutofdomain: Specifies a function isoutofdomain(t,u) where, when it returns false, it will reject the timestep. Defaults to always false.

One of them is wrong.

Reply:
Oh, that should say "when it returns true"